logstash-core/spec/logstash/pipeline_spec.rb

```diff
@@ -15,6 +15,21 @@ def close
   end
 end
 
+class DummyInputGenerator < LogStash::Inputs::Base
```

Comment:
I was thinking to use the generator but I have checked the `logstash-core` gemspec and it doesn't include the generator plugin and the only test in logstash-core that require the `generator` is the `pipeline_reporter_spec`. 

Its not obvious that the generator is available in the context of theses specs :disappointed: